website/content/userguide/qemu.md

```diff
@@ -0,0 +1,102 @@
+---
+title: "QEMU x86-64 virtual machine"
+menuTitle: "QEMU x86-64 virtual machine"
+weight: 70
+---
+
+In addition to running gokrazy on a Raspberry Pi or another ARM board, you can also run it in a QEMU x86-64 virtual
+machine. The following guide covers what's necessary to build and run a gokrazy image on QEMU. Your hypervisor may require additional configuration.
+
+## Before you begin
+
+If you don't already have Go installed, [install the latest version](https://golang.org/dl/). After that, install the `gok` command line tool:
+
+```bash
+go install github.com/gokrazy/tools/cmd/gok@main
+```
+
+## Building the image
+
+### Create a new gokrazy instance
+
+```bash
+gok new -i gok-vm
+```
+
+The `-i` flag specifies the name of the instance. In this case, we're using `gok-vm`, but you can use any name you like. The instance configuration will be created in your home directory under `~/gokrazy/gok-vm`.
+
+### Edit the instance configuration
+
+Open the instance configuration file in your favorite editor. The file is located at `~/gokrazy/gok-vm/config.json`. You'll need to make the following changes:
```

Comment:
Recommend `gok edit` instead of supplying the path please. Also, link “instance configuration” to https://gokrazy.org/userguide/instance-config/